config.go

```diff
@@ -218,6 +219,14 @@ func (h hypervisor) defaultBridges() uint32 {
 	return h.DefaultBridges
 }
 
+func (h hypervisor) blockDeviceDriver() string {
+	if h.BlockDeviceDriver == "" {
+		return defaultBlockDeviceDriver
+	}
+
+	return h.BlockDeviceDriver
```

Comment:
You should validate the block device  `virtio-scsi or virtio-blk`

Reply:
Done